src/config/SKYSTARSH7HD/config.h

```diff
@@ -29,6 +29,7 @@
 #define USE_GYRO
 #define USE_ACC
 #define USE_ACCGYRO_BMI270
+#define USE_BARO_BMP280
```

Comment:
Please move line 132 (#define USE_BARO) above this line

Reply:
ty!